SimG4Core/Geometry/src/DDG4ProductionCuts.cc

```diff
@@ -16,6 +17,12 @@
 
 using geant_units::operators::convertCmToMm;
 
+#ifdef HAVE_GEANT4_UNITS
+#define MM_2_CM 1.0
+#else
+#define MM_2_CM 0.1
+#endif
+
```

Comment:
Are these lines 20-25 just for temporary testing? I think we would use `dd4hep::mm`, so line 236 would look like:
```
double gammacut = spec->dblValue("ProdCutsForGamma") / dd4hep::mm;  // Convert from DD4hep units to mm
```

Reply:
@cvuosalo - these lines are indeed temporary and should be completely removed when we've updated to the new version of DD4hep with G4 units as default. I don't think we need to introduce any other operations in SimG4 where G4 units are the default ones.

Reply:
If we are going to remove these operations soon, it would be simpler to write:
```
double gammacut = spec->dblValue("ProdCutsForGamma") * 10.;  // FIXME: Remove conversion to mm when DD4hep goes to Geant4 units.
```
I don't like using "MM_2_CM", which seems to say "convert mm to cm", when we are actually converting cm to mm.

Reply:
`/ MM_2_CM` string says `inverse of "convert mm to cm"` since there is a division. It is also much easier to auto replace then 10. It’s intentionally chosen not to be liked to encourage the units replacement 😀

Reply:
Also as a first step in the units migration one would simply set `HAVE_GEANT4_UNITS` without code modifications.